docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+# CVE Creation Guide
+This is a guide to creating a CVE request via Mitre.
+It may be of some use to explain what the CVE fields are looking for with some hints and examples.
+
+# Creating a CVE Request
+
+This is an example of how to create a CVE with an older vulnerability from Mojolicious that did not have one previously assigned.
```

Comment:
Should we use a general example like `Foobar` or something like that, rather than Mojolicious?

Reply:
I can change it it an example but likely should be generic

Reply:
@stigtsp the thing I like about using this example is that it exists and you can review the source info

Reply:
@stigtsp I looked at it but the example is likely best.  The point was to give an example that someone could follow

Reply:
> Should we use a general example like `Foobar` or something like that, rather than Mojolicious?

Then please use Foo::Bar or Foo-Bar to make the distinction clear in what to use

Reply:
I use Foo-Bar in the security policy guide.